test/src/e2e_vm_tests/test_programs/ec_recover_test/Forc.toml

```diff
@@ -5,5 +5,6 @@ name = "ec_recover_test"
 entry = "main.sw"
 
 [dependencies]
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
```

Comment:
Why are you duplicating this?

Reply:
I didn't touch the file myself, investigating and fixing

Reply:
I probably messed up my merge at some point, I'm too used to rebasing I lost my touch for merging haha